docroot/themes/custom/uids_base/templates/views/views-view-unformatted--person-card.html.twig

```diff
@@ -0,0 +1,116 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a view of unformatted rows.
+ *
+ * Available variables:
+ * - title: The title of this group of rows. May be empty.
+ * - rows: A list of the view's row items.
+ *   - attributes: The row's HTML attributes.
+ *   - content: The row's content.
+ * - view: The view object.
+ * - default_row_class: A flag indicating whether default classes should be
+ *   used on rows.
+ *
+ * @see template_preprocess_views_view_unformatted()
+ *
+ * @ingroup themeable
+ */
+#}
+{% if title %}
+  <h3>{{ title }}</h3>
+{% endif %}
+{% for key,row in rows %}
+  {% set row_classes = [
+    default_row_class ? 'views-row',
+  ] %}
+  <div{{ row.attributes.addClass(row_classes) }}>
+
+    {% set media_attributes = media_attributes ?: create_attribute() %}
+
+    {% set person_card_view = {
+      'card_image': view.style_plugin.getField(key, 'field_image'),
+      'card_subtitle': view.style_plugin.getField(key, 'field_person_email'),
+      'card_tag': view.style_plugin.getField(key, 'field_person_phone'),
+      'person_website': view.style_plugin.getField(key, 'field_person_website'),
+      'person_education': view.style_plugin.getField(key, 'field_person_education'),
+      'person_hometown': view.style_plugin.getField(key, 'field_person_hometown'),
+      'person_contact': view.style_plugin.getField(key, 'field_person_contact_information'),
+      'media_attributes': media_attributes.addClass('media--circle card__media--large card__media'),
+    } %}
+
+    {%
+      set classes = [
+      'card--stacked',
+      'card--centered-left',
+      'card--alignment-left',
+      'media--border',
+    ]
+    %}
+
+    {% set has_body = false %}
+
+    {% set card_body_fields = ['card_subtitle', 'card_tag', 'person_website', 'person_education', 'person_hometown', 'person_contact'] %}
+
+    {% for check in card_body_fields %}
+      {% if person_card_view[check]|striptags|trim is not empty %}
+        {{ check }}
```

Comment:
Is this still needed for the last todo items, or can it be taken out?

Reply:
It is stripped in a commit that I have not pushed yet.